templates/explore/repo_list.tmpl

```diff
@@ -3,10 +3,10 @@
 		<div class="item">
 			<div class="ui header">
 				<a class="name" href="{{AppSubUrl}}/{{if .Owner}}{{.Owner.Name}}{{else if $.Org}}{{$.Org.Name}}{{else}}{{$.Owner.Name}}{{end}}/{{.Name}}">{{if $.PageIsExplore}}{{.Owner.Name}} / {{end}}{{.Name}}</a>
-				{{if .IsPrivate}}
-					<span class="text gold"><i class="icon octicon octicon-lock"></i></span>
-				{{else if .IsFork}}
+				{{if .IsFork}}
```

Comment:
I think should still show its private first.

We can add a small line to say where it forks from, but after changed this, we can't tell it is private or not.